packages/graphql/src/translate/queryAST/ast/filters/authorization-filters/AuthorizationFilters.ts

```diff
@@ -47,9 +48,9 @@ export class AuthorizationFilters extends QueryASTNode {
         return Cypher.or(...this.filters.map((f) => f.getPredicate(context)));
     }
 
-    public getValidation(context: QueryASTContext): Cypher.VoidProcedure | undefined {
+    public getValidation(context: QueryASTContext, when: ValidateWhen = "BEFORE"): Cypher.VoidProcedure | undefined {
```

Comment:
```suggestion
    public getValidation(context: QueryASTContext, when: ValidateWhen): Cypher.VoidProcedure | undefined {
```
I think it's better keep it without the default value, I believe by having explicitly defined it's more visible and less error prone. However, up to you if this requires too much work, as the change itself it's already out of the initial scope.

Reply:
Usually yes, but `BEFORE` and `AFTER` only really apply on mutations (in fact, only on some of them!) so it would require explicitly setting this value in places where, realistically, shouldn't even be configurable